### PR TITLE
7 add analytics

### PR DESCRIPTION
**Pull request by debonx**

### CODE REVIEW

**Comment by debonx** [comment]
- [ ] Privacy Policy needs to be visible even if cookies are not accepted

File: components/cookie-consent/cookie-consent.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  useEffect(() => {
+    const cookieConsent = getCookie(COOKIE_CONSENT_KEY)
+    if (cookieConsent) {
+      setCookieConsent(true)
+    } else {
+      setCookieConsent(false)
+    }
+  }, [])
+
+  if (cookieConsent) {
```
Comment (debonx) [inline components/cookie-consent/cookie-consent.tsx:19]: - [ ] Privact policy needs to be visible without accepting cookies

File: components/cookie-consent/cookie-consent.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    return null
+  }
+
+  const handleClick = (
```
Comment (silversonicaxel) [inline components/cookie-consent/cookie-consent.tsx:23]: What about using `useCallback` ?

File: yarn-error.log
Comment (silversonicaxel) [inline yarn-error.log:1]: I would remove this file.

**Review by silversonicaxel — APPROVED** [review]
It seems legit. (Tomorrow I'll send you a whatsapp msg)

File: components/cookie-consent/cookie-consent.tsx (hunk 2 above)
Comment (debonx) [reply components/cookie-consent/cookie-consent.tsx:23]: Uhuh, nice to see you around. Let me add that indeed :)

**Pull request merged**